### ISSUE TITLE
When code block loads the caption is missing

### ISSUE THREAD

**Issue by bobbyzholmes** [neyre/sir-trevor-wp#3]
The following code doesn't load the caption in the code block
```
loadData: function(data){
		this.getTextBlock().html(SirTrevor.toHTML(data.text, this.type));
	}
```

I fixed the issue with the following

```
loadData: function(data){
		this.getTextBlock().html(SirTrevor.toHTML(data.text, this.type));
		this.$('.js-caption-input').val(data.caption);
	}
```

**Comment by neyre** [comment]
Fixed in b8a1902

**Comment by mcshanea** [comment]
Many thanks @bobbyzholmes (and @neyre) ...we were missing using this plugin over on tech.eu!
